README.md

```diff
@@ -17,21 +17,21 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[x] Strings
+[x] Booleans
+[x] Undefined
 [] NaN
 [] Integers
 [] Arrays
-[] Null
+[x] Null
 ```
```

Comment:
:+1: Something minor to add, Integers are a type of `Number` which is a primitive data type in JS. Also, `Arrays` are an example of a complex data type in JS